doc/admin-guide/plugins/tcpinfo.en.rst

```diff
@@ -117,15 +117,15 @@ The following options may be specified in :file:`plugin.config`:
   settings discussed below.
 
 --rolling-offset-hr=VALUE
-  Set the hour ``0`` to ``23`` at which the ``tcp-metrics.log`` will roll when
+  Set the hour ``0`` to ``23`` at which the output log file will roll when
   using interval rolling. Default value is ``0``.
 
 --rolling-interval-sec=VALUE
-  Set the rolling interval in seconds for the ``tcp-metrics.log``. May be set 
+  Set the rolling interval in seconds for the output log file. May be set 
   from ``60`` to ``86400`` seconds, Defaults to ``86400``.
 
 --rolling-size-mb=VALUE
-  Set the size at which the ``tcp-metics.log`` will roll when using log size 
+  Set the size at which the output log file  will roll when using log size 
```

Comment:
You'll need to rename this after the `ink_aoui()` change. Also mention that this accepts size suffixes ("K", "M", "G", etc).